src/main/scala/is/hail/io/vcf/HtsjdkRecordReader.scala

```diff
@@ -62,149 +62,9 @@ class HtsjdkRecordReader(val callFields: Set[String]) extends Serializable {
     }
     rvb.endStruct() // info
   }
-
-  def readRecord(vc: VariantContext, rvb: RegionValueBuilder, infoType: TStruct, gType: TStruct, dropSamples: Boolean,
```

Comment:
This is still used by `LoadGDB`. I'm don't think you can delete this.

Reply:
loadGDB is gone